ui/app/components/global-search/control.js

```diff
@@ -111,7 +118,9 @@ export default class GlobalSearchControl extends Component {
   openOnClickOrTab(select, { target }) {
     // Bypass having to press enter to access search after clicking/tabbing
     const targetClassList = target.classList;
-    const targetIsTrigger = targetClassList.contains('ember-power-select-trigger');
+    const targetIsTrigger =
+      targetClassList.contains('ember-power-select-trigger') ||
+      targetClassList.contains('ember-power-select-search-input');
```

Comment:
This is my attempt to fix the search focus blackhole problem, I added this selector because was in the target class list when I clicked in the padding as indicated in this screenshot, which is the only way I was able to reproduce that problem:

<img width="444" alt="image" src="https://user-images.githubusercontent.com/43280/85568880-afd8ec00-b5f7-11ea-8f9f-c77d411fb359.png">

However, the problem is still happening for @DingoEatingFuzz and I’m again having problems reproducing so maybe this is not actually doing anything 😞